File: Desktop/SP/Paragraph.cs
public class Paragraph : Element
{
    private string text;

    public Paragraph(string text)
    {
        this.text = text;
    }

    public void print()
    {
        Console.WriteLine("Paragraph: " + text);
    }

    public void add(Element element)
    {
        throw new NotSupportedException();
    }

    public void remove(Element element)
    {
        throw new NotSupportedException();
    }

    public Element get(int index)
    {
        throw new NotSupportedException();
    }
}

File: src/main/java/org/example/book/Paragraph.java
package org.example.book;

import org.example.strategy.AlignStrategy;

public class Paragraph extends Element {
    private String text;
    private AlignStrategy alignStrategy;

    public Paragraph(String text) {
        this.text = text;
        this.alignStrategy = null;
    }

    public String getText() {
        return text;
    }

    public void setAlignStrategy(AlignStrategy alignStrategy) {
        this.alignStrategy = alignStrategy;
    }

    @Override
    public void print() {
        if (alignStrategy != null) {
            alignStrategy.render(this, null);
        } else {
            System.out.println("Alignment: ");
            System.out.println("Text: " + text);
        }
    }
}